recipes/trnanalysis/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "0.1.2" %}
+{% set version = "0.1.3" %}
 
 package:
   name: trnanalysis
   version: {{ version }}
 
 source:
-  url: https://files.pythonhosted.org/packages/94/55/d5d9dc753bf1454f379034ca1cb5c94b30827694b28d8c99bf4f5c95e9d2/tRNAnalysis-0.1.2.tar.gz
-  sha256: 66d442c6fb4f83382145f4d759bf60fc0087b8e6b6c9f9eb4f89e7e20d58a8db
+  url: https://files.pythonhosted.org/packages/2d/58/e36a3678a8a887f3c22780443c8bdcbeb1d3d01de882693490b562de886f/tRNAnalysis-0.1.3.tar.gz
+  sha256: 36e876633ccbd06a91e245c476a343918979775f2b9900b80bfd035a50b823c3
 
 build:
   skip: True # [not py==36]
```

Comment:
Do you know if this is still python 3.6 only?

Reply:
Many thanks for the restart.
Yes, its still python 3.6 only for the moment because cgat-apps is currently not correctly built for python 3.7  #(https://github.com/cgat-developers/cgat-apps/issues/43)